packages/superset-ui-chart/yarn.lock

```diff
@@ -0,0 +1,273 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
```

Comment:
We only use `yarn.lock` in the root directory. https://github.com/apache-superset/superset-ui/pull/362#discussion_r408291488

I'm creating a PR to add `yarn.lock` to `.gitignore`.

Reply:
oops, thanks